Kconfig

```diff
@@ -50,6 +50,27 @@ endchoice
 
 endmenu
 
+menu "IMU configuration"
+
+choice
+    prompt "IMU algorithm"
+    default IMU_MAYHONY_QUATERNION
+
+config IMU_MADGWICK_QUATERNION
+    bool "Madgwick's AHRS and IMU algorithms"
+    help
+        Use Madgwick's IMU and AHRS algorithms.
+        See: http://www.x-io.co.uk/open-source-ahrs-with-x-imu
+
+config IMU_MAYHONY_QUATERNION
+    bool "Madgwick's implementation of Mayhony"
```

Comment:
To avoid confusion, just call this Mayhony and refer to the paper they refer too? https://ieeexplore.ieee.org/document/4608934

Reply:
Agreed! I pushed a commit to fix it up!